### PR TITLE
Use SSH root password provided by user

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: caasp-stack.yaml [hunk 1]
```diff
@@ -336,7 +336,7 @@ resources:
 
                 chpasswd:
                   list: |
-                    root:linux
+                    root:$root_password
```
Comment (kiall) [inline caasp-stack.yaml:339]: `$root_password` would need to be added in the params section too.

**Review by kiall — CHANGES_REQUESTED** [review]

File: caasp-stack.yaml (hunk 1 above)
Comment (mjura) [reply caasp-stack.yaml:339]: It is already there https://github.com/SUSE/caasp-openstack-heat-templates/blob/master/caasp-stack.yaml#L68

**Review by sayalilunkad — DISMISSED** [review]

File: caasp-stack.yaml (hunk 1 above)
Comment (kiall) [reply caasp-stack.yaml:339]: It needs to go here: https://github.com/SUSE/caasp-openstack-heat-templates/blob/master/caasp-stack.yaml#L345

Comment (mjura) [reply caasp-stack.yaml:339]: Ahhh, I'm sorry,  you are right. I fixed it

**Review by kiall — APPROVED** [review]

**Pull request merged**